nemo/utils/exp_manager.py

```diff
@@ -238,6 +289,11 @@ def exp_manager(trainer: 'pytorch_lightning.Trainer', cfg: Optional[Union[DictCo
             cfg.wandb_logger_kwargs,
         )
 
+    # add loggers timing callbacks
+    if cfg.log_step_timing:
+        timing_callback = TimingCallback(timer_kwargs=cfg.step_timing_kwargs)
```

Comment:
Since kwargs are optional in the data class, check for none here and replace with {}

Reply:
Good point. Replaced with `cfg.step_timing_kwargs or {}`.